code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm

```diff
@@ -8,54 +8,51 @@
+			var/mob/living/M = buck
 
-/obj/structure/stool/bed/nest/user_unbuckle_mob(mob/living/user)
-	if(buckled_mob && buckled_mob.buckled == src)
-		var/mob/living/M = buckled_mob
-
-		if(user.get_int_organ(/obj/item/organ/internal/xenos/plasmavessel))
-			unbuckle_mob()
-			add_fingerprint(user)
-			return
-
-		if(M != user)
-			M.visible_message(\
-				"<span class='notice'>[user.name] pulls [M.name] free from the sticky nest!</span>",\
-				"<span class='notice'>[user.name] pulls you free from the gelatinous resin.</span>",\
-				"<span class='notice'>You hear squelching...</span>")
-
-		else
-			buckled_mob.visible_message(\
-				"<span class='warning'>[buckled_mob.name] struggles to break free of the gelatinous resin...</span>",\
-				"<span class='warning'>You struggle to break free from the gelatinous resin... (This will take around 2 minutes and you need to stay still)</span>",\
-				"<span class='notice'>You hear squelching...</span>")
-			if(!do_after(M, 1200, target = src))
-				if(M && M.buckled)
-					to_chat(M, "<span class='warning'>You fail to escape \the [src]!</span>")
+			if(user.get_int_organ(/obj/item/organ/internal/xenos/plasmavessel))
+				unbuckle_mob(M)
+				add_fingerprint(user)
 				return
-			if(!M.buckled)
-				return
-			M.visible_message(\
-			"<span class='warning'>[M.name] breaks free from the gelatinous resin!</span>",\
-			"<span class='notice'>You break free from the gelatinous resin!</span>",\
-			"<span class='italics'>You hear squelching...</span>")
-		unbuckle_mob()
-		add_fingerprint(user)
 
+			if(M != user)
+				M.visible_message(\
+					"[user.name] pulls [M.name] free from the sticky nest!",\
+					"<span class='notice'>[user.name] pulls you free from the gelatinous resin.</span>",\
+					"<span class='italics'>You hear squelching...</span>")
+			else
+				M.visible_message(\
+					"<span class='warning'>[M.name] struggles to break free from the gelatinous resin!</span>",\
+					"<span class='notice'>You struggle to break free from the gelatinous resin... (Stay still for two minutes.)</span>",\
+					"<span class='italics'>You hear squelching...</span>")
+				if(!do_after(M, 1200, target = src))
+					if(M && M.buckled)
+						M << "<span class='warning'>You fail to unbuckle yourself!</span>"
+					return
+				if(!M.buckled)
+					return
+				M.visible_message(\
+					"<span class='warning'>[M.name] breaks free from the gelatinous resin!</span>",\
+					"<span class='notice'>You break free from the gelatinous resin!</span>",\
+					"<span class='italics'>You hear squelching...</span>")
+
+			unbuckle_mob(M)
+			add_fingerprint(user)
 
 /obj/structure/stool/bed/nest/user_buckle_mob(mob/living/M, mob/living/user)
-	if( !ismob(M) || (get_dist(src, user) > 1) || (M.loc != src.loc) || user.restrained() || usr.stat || M.buckled || istype(user, /mob/living/silicon/pai) )
+	if ( !ismob(M) || (get_dist(src, user) > 1) || (M.loc != src.loc) || user.incapacitated() || M.buckled )

```

Comment:
Why remove user.restrained()?

Reply:
its a catch all https://github.com/ParadiseSS13/Paradise/blob/3eaff16f45583bd60366e96fdbc6bd63f29dc9c4/code/modules/mob/living/update_status.dm#L59

Reply:
better question..why did i add that fucking space on the if...